### PR TITLE
Add common modal files

### PR DESCRIPTION
**Pull request by jlight99**
- Add a common modal component that will be used by future modal components
- Add common modal styling

### CODE REVIEW

**Comment by guy-ro** [comment]
Hey Ellen!

Check out https://github.com/uwblueprint/pregnancy-centre/pull/44
There's some common styling for buttons here that you can use

EDIT: That PR is now merged!

**Review by meganniu — APPROVED** [review]
LGTM!!! the styling looks beautiful

File: client/src/components/organisms/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import '../../pages/Modal.scss';
+import React, { useState } from 'react';
+import { useApolloClient } from "@apollo/client";
+import { useDispatch } from "react-redux";
+
+import { Button, Modal } from 'react-bootstrap';
+
+import tpcLeaf from '../../assets/tpc-leaf.png';
+
+function CommonModal(props: any) {
```
Comment (meganniu) [inline client/src/components/organisms/Modal.tsx]: nitpick: could we type `props` using an interface?

File: client/src/pages/Modal.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.modal {
```
Comment (meganniu) [inline client/src/pages/Modal.scss]: could we move `Modal.scss` to `components/organisms/style`? since `Modal.tsx` is in `components/organisms`

**Review by Anand-Issac — APPROVED** [review]
Lgtm

**Pull request merged**